yarn.lock

```diff
@@ -1369,6 +1369,31 @@
     magic-string "^0.25.2"
     resolve "^1.11.0"
 
+"@rollup/plugin-commonjs@^17.0.0":
```

Comment:
A recent change by @katelynsills reversed yarn.lock changes that look very similar to these. You might `yarn install --force` to make sure these are not flaps.

Reply:
syncing rollup dev dependencies to versions used in other packages makes this yarn lockfile change go away